samples/sampler/stories/default/TabGroup.js

```diff
@@ -13,8 +14,10 @@ export default {
 
 export const _TabGroup = (args) => {
 	const orientation = args['orientation'];
+
 	return (
 		<TabGroup
+			onSelect={action('onSelect')}
```

Comment:
The qa-a11y sample also has a TabGroup. Could you please fix qa-a11y sampler also?

Reply:
Can you add the "selectedIndex" prop to make show the selected state? The selection style is clear on the Carbon skin.